CMakeLists.txt

```diff
@@ -464,7 +464,7 @@ set(OFFSETS_O_PATH ${CMAKE_CURRENT_BINARY_DIR}/CMakeFiles/offsets.dir/arch/${ARC
 set(OFFSETS_H_PATH ${PROJECT_BINARY_DIR}/include/generated/offsets.h)
 
 add_library(          offsets STATIC ${OFFSETS_C_PATH})
-target_link_libraries(offsets zephyr_interface)
+target_link_libraries(offsets zephyr_interface -u_OffsetAbsSyms -u_ConfigAbsSyms )
```

Comment:
AFAICT _ConfigAbsSyms is not related to 'offsets', it is related to configs.c.

Reply:
you're right.
and configs.c is a generated file which is placed inside libzephyr.a, which means the flag should go to zephyr lib and not even zephyr_interface.
Cause the flag is needed when you link to zephyr lib, not because you use zephyr_interface.
Other libraries should know about this flag during compilation, the flag is only required when linking the zephyr lib.

Example:
add_library(MyLib ....)
target_link_libraries(MyLib zephyr_interface)
add_executable(MyProgram)
target_link_libraries(MyProgram MyLib) # The flag is not needed cause libzephyr.a is not linked in this case.

but doing:
target_link_libraries(MyProgram MyLib zephyr) # Requires the flag when linking to zephyr, hence, Zephyr lib is the correct place for the flag